3rdparty/protobuf/googleapis/README.md

```diff
@@ -0,0 +1,6 @@
+This is a dump of the .proto files from https://github.com/googleapis/googleapis directory google.
+
+This dump was taken at git sha e17dbfb19652240490cae8adeb89991d13cf9df7.
+
+It contains many more protos than we actually use, but performing dependency analysis to work out
```

Comment:
So, I could understand this argument if it were a dozen files or so, but for a few hundred it doesn't quite jive for me. In particular, the cost continues after the initial commit in that whenever we bump/update these files, we'll have hundreds of edits to things we're not using.

Are you expecting that the number of "live" files here is going to be more than a dozen or so? If not, it would be good to do something like "delete all but the root, and re-add things one by one until it compiles".

Reply:
Turns out the deps were actually pretty small. Done.